test/index.test.ts

```
@@ -0,0 +1,58 @@
+  });
+
+  it('should callback work', done => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    function fn2(data: any) {
+      assert.deepEqual(data, { foo: 'bar\u2029-ok\u2028' });
+      done();
+    }
+    // eslint-disable-next-line no-eval
+    eval(jsonp({ foo: 'bar\u2029-ok\u2028' }, 'fn2'));
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)